### PR TITLE
Add a README

### PR DESCRIPTION
**Pull request by stefanberger**
Add install and uninstall targets to the Makefile and then add a README to the project.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
+# imgcrypt image encryption library and command line lool
+
+The `imgcrypt` library provides API exensions for containerd to support encryted container images and implements
+the `ctd-decoder` command line tool for use by containerd to decrypt encrypted container images. An extended version
+of containerd's `ctr` tool with support for encrypting and decrypting container images is also provided.
```
Comment (lumjjb) [inline README.md]: I'm thinking if we should set the name for the `ctr` tool to `ctr-enc` to signify that it is encryption supported `ctr`. In addition, we don't want to create confusion or issues with people replacing their `ctr` binaries and then having confusion when moving between developing for containerd and working with encryption.

File: Makefile [hunk 2]
```diff
@@ -12,25 +12,45 @@
...
+
+all: build
+
+build: $(BINARIES)
 
 FORCE:
 
-decoder: FORCE
+ctd-decoder: FORCE
 	go build -v ./cmd/ctd-decoder/
```
Comment (lumjjb) [inline Makefile]: suggestion: what do you think about a `bin/` directory for build output

Comment (stefanberger) [reply Makefile]: Yes, fixed.

File: README.md (hunk 1 above)
Comment (stefanberger) [reply README.md]: I renamed it to `ctr-enc` now.

**Pull request merged**